loadDocument.js

```diff
@@ -0,0 +1,33 @@
+const readFiles = require('./readFiles');
+const CVSparser = require('./CVSparser');
+const JSONparser = require('./JSONparser');
+const XMLparser = require('./XMLparser');
+const loggerMessages = require('./loggerMessages');
+const TransactionsLib = require('./TransactionsLib');
+
+module.exports = function(fileName, logger) {
+
+    const rawFile = readFiles(fileName, logger);
+    
+    if (fileName.substr(-3) === 'csv') {

```

Comment:
Taking the parsed file + creating the transactions happens in all cases; would be good to have one function return the raw data so that then whatever data you get you just create transactions from it.